justfile

```diff
@@ -1,6 +1,82 @@
-# Setup the tools needed to develop
-setup-tools:
-    cargo install cargo-release insta 
-# publish rust crates    
-release-crates:
+#!/usr/bin/env -S just --justfile
```

Comment:
this is used for run justfile directly like `./justfile` ?
do you add execute permission for this file?